docs/devops.md

```diff
@@ -957,19 +957,19 @@ We use [a CLI tool](https://github.com/freecodecamp/sendgrid-email-blast) to sen
 
 7. When the email blast is complete, verify that no emails have failed before destroying the droplets.
 
-# Flight Manual - Adding news instances for new languges
+# Flight Manual - Adding news instances for new languages
 
 ### Theme Changes
 
 We use a custom [theme](https://github.com/freeCodeCamp/news-theme) for our news publication. Adding the following changes to the theme enables the addition of new languages.
 
-1. Include the an else if statment for the new [ISO language code](https://www.loc.gov/standards/iso639-2/php/code_list.php) in [setup-local.js](https://github.com/freeCodeCamp/news-theme/blob/main/assets/config/setup-locale.js)
-2. Create an initial config folder by duplicating the [assets/config/en](https://github.com/freeCodeCamp/news-theme/tree/main/assets/config/en) folder and changing its name to the new language code. (en—> es for Spanish)
-3. Inside the new language folder, change the variable names in main.js and footer.js to the relevant language short code (enMain —> esMain for Spanish)
-4. Duplicate the [locals/en.json](https://github.com/freeCodeCamp/news-theme/blob/main/locales/en.json) and rename it to the new language code.
-5. In [partials/i18n.hbs](https://github.com/freeCodeCamp/news-theme/blob/main/partials/i18n.hbs), add scripts for newly created config files.
-6. Add the related language day.js script from [cdnjs](https://cdnjs.com/libraries/dayjs/1.10.4) to [freecodecamp cdn](https://github.com/freeCodeCamp/cdn/tree/main/build/news-assets/dayjs/1.10.4/locale)
+1. Include an else if statment for the new [ISO language code](https://www.loc.gov/standards/iso639-2/php/code_list.php) in [`setup-local.js`](https://github.com/freeCodeCamp/news-theme/blob/main/assets/config/setup-locale.js)
+2. Create an initial config folder by duplicating the [`assets/config/en`](https://github.com/freeCodeCamp/news-theme/tree/main/assets/config/en) folder and changing its name to the new language code. (`en` —> `es` for Spanish)
+3. Inside the new language folder, change the variable names in `main.js` and `footer.js` to the relevant language short code (`enMain` —> `esMain` for Spanish)
+4. Duplicate the [`locales/en.json`](https://github.com/freeCodeCamp/news-theme/blob/main/locales/en.json) and rename it to the new language code.
```

Comment:
```suggestion
4. Duplicate the [`locales/es.json`](https://github.com/freeCodeCamp/news-theme/blob/main/locales/en.json) and rename it to the new language code.
```

The locales/en.json doesn't have all translatable strings

Reply:
Well, this does not fix the link either.

Does this mean we should rather update the `en.json` file?

Reply:
wdym that it doesn't fix the link?

I don't know why en.json doesn't have all strings, maybe @scissorsneedfoodtoo knows

Reply:
Yes, we should probably follow our standard flow of using the english file as the single source of truth.

Reply:
> wdym that it doesn't fix the link?

Before your edit, the text was updated, but not the file link.

Reply:
Hey all, sorry, just seeing this now.

Currently the `en.json` file doesn't have all the strings because we're mostly using the English fallback method of translating. So anything in the `en.json` file is there because we're using generic keys for those particular strings.

Right now, `es.json` is one of the more complete files. `zh.json` was originally based on `es.json`, but I removed some Twitter related strings we're not using for the social banner.

It would be better to move over to generic keys for everything, though. This would make the `locale.json` files more consistent, and help keep things synchronized with 11ty localization.

Reply:
So we should leave this as pointing to `en.json` and update the file in the news theme?

Reply:
@nhcarrigan, I think you're right. That would be easier overall.

I'll make a quick PR to update the `en.json` file in the news theme to include all the strings. There will be a lot of repetition, but it's not a big deal since this is more of an internal process.

Later we can update the News theme PR to use generic keys like we do in the main repo and 11ty. Once that's done, we won't need to come back and update these docs again.